### PR TITLE
created Gripper

### PR DESCRIPTION
**Pull request by Spikes-2212**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Gripper.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    private static Gripper instance;
+
+    private final DoubleSolenoid solenoid;
+
+    private Gripper(String namespaceName, DoubleSolenoid solenoid) {
+        super(namespaceName);
+        this.solenoid = solenoid;
+    }
+
+    public static Gripper getInstance() {
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/subsystems/Gripper.java:14]: move this to be above the constructor

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]
Make this compile

File: src/main/java/frc/robot/subsystems/Gripper.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+public class Gripper extends DashboardedSubsystem {
+
+    private static Gripper instance;
+
+    private final DoubleSolenoid solenoid;
+
+    private Gripper(String namespaceName, DoubleSolenoid solenoid) {
+        super(namespaceName);
+        this.solenoid = solenoid;
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/subsystems/Gripper.java]: add a call to configureDashboard in the end of the constructor

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/CloseGripper.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.Gripper;
+
+public class CloseGripper extends InstantCommand {
+
+    public CloseGripper(Gripper gripper) {
+        addRequirements(gripper);
+        gripper.closeGripper();
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/CloseGripper.java]: why is this in the constructor?

File: src/main/java/frc/robot/commands/OpenGripper.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.Gripper;
+
+public class OpenGripper extends InstantCommand {
+
+    public OpenGripper(Gripper gripper) {
+        addRequirements(gripper);
+        gripper.openGripper();
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/OpenGripper.java]: why is this in the constructor?

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

**Review by OfriRosenbaum — APPROVED** [review]

**Review by Yoel-Perman-Brilliant — APPROVED** [review]

**Pull request merged**